grr/server/grr_response_server/bin/grrafana_test.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
+# Lint as: python3
+"""Unittest for GRRafana HTTP server."""
+from grr.test_lib import test_lib
+from grr_response_server.bin import grrafana
+
+# TODO(tsehori): We should implement tests for grrafana.py.
+class GRRafanaTest(test_lib.GRRBaseTest):
```

Comment:
Please, avoid using `GRRBaseTest` unless there is a compelling reason to do so. `absltest.TestCase` is enough, especially that this is empty right now.